controllers/ApiController.php

```diff
@@ -73,6 +74,10 @@ public function init($extra) {
 			$this->e503(Z_CONFIG::$MAINTENANCE_MESSAGE);
 		}
 		
+		if (Z_CONFIG::$BACKOFF > 0) {
```

Comment:
`if (!empty(Z_CONFIG::$BACKOFF)) {`

(We should allow optional/new parameters to not exist without logging a notice when possible.)